### PR TITLE
Peadiatric Hospital Website

### PR DESCRIPTION
**Pull request by kirtimahadane**
# Your Name
Kirti Parag Mahadane
## Submission checklist

- [x] Your submission follows best practices for [commit messages](https://chris.beams.io/posts/git-commit/) AND for [pull requests](https://github.community/t/best-practices-for-pull-requests/10195)
- [x] `Steps to run the project` AND a `documentation` have been included in a README.md file at root of your project.
- [x] No `binaries/compressed` files have been added
- [x] All pre-existing files in the repository have been removed
- [x] `Screenshots` have been added to the screenshots folder (optional for backend code).
- [x] All italicesed instructions under each submission heading inline, have been removed.
- [x] You understand that a submission here is publicly visible. 
- [x] You have not plagialised, or blatently copied work; and this submission is your original work. (Code of ethics)

## Briefly write about the project that you have submitted from the perspective of the user.
This website is for paediatric hospital.The site is under construction.It has 4 pages namely 
home, about, departemnts and contact. The website looks good on all devices i.e. desktop, tablet and mobile as well.

## Assumptions you have made for this project?
No assumptions.

## Other information (like testing credentials)
 Check responsive behaviour of website.

## Did you learn anything new while doing this assignment? Please explain.
I have used css blend-mode property for the first time. 
I am using github for such activity first time. 

## How much time did it take for you complete the project?
It took 2 days time to complete the project. I had images and content ready. Normally, form concept to design ,
 this project will take atleast 5 days to complete.
## If you had more time, what enhancements will you make?
I would have worked on mobile navigation, footer layout and color scheme.

## What was your motivation behind choosing this evaluation mode and the project?
To showcase my front end design ability to create responsive , clean and functional code.

### CODE REVIEW

File: departments.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+        
+          
+       
+         
+      </div>
+    </div>
+  </div>
+  
+  </section>
+<section id="dept-main">
```
Comment (pradnyalgandhi) [inline departments.html:65]: The cards do not seem to be responsive when the browser window is resized.

**Review by pradnyalgandhi — CHANGES_REQUESTED** [review]
Thank you for your submission. Can you address the following changes : 
- Instead of using Bootstrap for styling, can you try using pure CSS for styling.
- Make the webpage responsive using media queries.

**Comment by kirtimahadane** [comment]
Hello Pradnyal,
Do you want me to do changes with the entire site OR only departments page?
I will have to check accordingly.
Kirti.

**Comment by kirtimahadane** [comment]
Yes , the problem may be due to transform property.
I will need 2 days time to fix it.

File: departments.html (hunk 1 above)
Comment (kirtimahadane) [reply departments.html:65]: Departments page responsive. 
Please check.

**Comment by pradnyalgandhi** [comment]
Thank you for your submission. We have dropped you a message in private.

**Pull request closed without merging**